storage/test/connector.cc

```diff
@@ -58,8 +58,8 @@ TEST(StorageFactory, Factory) {
 }
 
 TEST(StorageFactory, FactoryWithFullConf) {
-  database_config dbcfg("MySQL", "127.0.0.1", 3306, "centreon", "centreon",
-                        "centreon_storage", 5, true, 5);
+  database_config dbcfg("MySQL", "127.0.0.1", "/var/lib/mysql/mysql.sock", 3306,
```

Comment:
And here, we could set the socket to "" since, 127.0.0.1 forces a tcp connection.